wp-content/themes/eighshift-task/src/blocks/components/paragraph/paragraph-style.scss

```diff
@@ -3,6 +3,36 @@ $paragraph: (
     default: $base-black-color,
     primary: $base-primary-color,
   ),
+  sizes: (
+    mobile: (
+      default: (
+        font-size: 16px,
+        line-height: 1,
+      ),
+      medium: (
+        font-size: 16px,
+        line-height: 1,
+      ),
+      big: (
+        font-size: 18px,
+        line-height: 1,
+      ),
+    ),
+    tablet: (
+      default: (
```

Comment:
if the size is the same no need to write it

Reply:
👍